docker/datahub-gms/start.sh

```diff
@@ -45,7 +40,7 @@ fi
 dockerize \
   -wait tcp://$EBEAN_DATASOURCE_HOST \
   -wait tcp://$(echo $KAFKA_BOOTSTRAP_SERVER | sed 's/,/ -wait tcp:\/\//g') \
-  -wait $ELASTICSEARCH_PROTOCOL://$ELASTICSEARCH_HOST_URL:$ELASTICSEARCH_PORT -wait-http-header "$ELASTICSEARCH_AUTH_HEADER" \
+  -wait $ELASTICSEARCH_PROTOCOL://$ELASTICSEARCH_HOST:$ELASTICSEARCH_PORT -wait-http-header "$ELASTICSEARCH_AUTH_HEADER" \
```

Comment:
Is changing ELASTICSEARCH_HOST_URL to ELASTICSEARCH_HOST going to break anyone?

Reply:
So ELASTICSEARCH_HOST is the exposed env variable and ELASTICSEARCH_HOST_URL was a processed variable. Now that we don't have to add username and password to it, there is no need for this variable.